test/e2e/setup.sh

```diff
@@ -121,6 +121,7 @@ helm install hub-agent ../../charts/hub-agent/ \
     --set logVerbosity=5 \
     --set enableWebhook=true \
     --set webhookClientConnectionType=service \
+    --set forceDeleteWaitTime="2m0s" \
```

Comment:
nit: can this duration be shorter? The longer this is, the more wait the forcedelete e2e needs